packages/odyssey-react/src/components/List/List.theme.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  DescriptionGridGapColumns: theme.SpaceRemM,
+  DescriptionGridGapRows: theme.SpaceRemXs,
+  MarginBlockEnd: theme.SpaceRemM,
+  MarginBlockStart: 0,
+  MarginInline: 0,
+  MaxWidth: theme.FontLineLengthMax,
+  OrderedUnorderedPaddingInlineStart: theme.SpaceRemM,
```

Comment:
one of the things I like about adding these name "parts" as a suffix (as opposed to a prefix here) is that they occupy the same part of the variable name that variants do - so all the tokens become more uniform and consistent across the API!

`SpacePaddingInlineStartOrderedUnordered`

Reply:
updated in e9bb90c

Reply:
The uniformity and consistency come at a cost to developers who are concerned primarily with the style of a particular element.

placing the element name as the suffix makes it maximally difficult to locate.

I do not believe that there is a necessity or much benefit to consistency of naming across token levels.

`OrderedUnorderedPaddingInlineStart: theme.SpaceRemM` is both readable and understandable

Reply:
@andrewberg-okta good feedback, my thinking is that it might be difficult to locate only until the pattern of the names becomes familiar. And since the pattern would be consistent across both global tokens and component variables we would reduce the cognitive overhead of switching the naming scheme as you're targeting one type of value or another within the theme API. That consistency helps the API feel more natural even as we iterate on the names to make them better or more discoverable.